### ISSUE TITLE
Apply hint: eta reduce breaks where block identation

### ISSUE THREAD

**Issue by jneira** [mpickering/apply-refact#94]
Hi, using hls-hlint-plugin an user has detected that apply "eta reduce" removes the identation of a where block, making the code throw parser errors

```shell
PS D:\dev\ws\haskell\cabal-test> cat .\src\HlintTests.hs
module HlintTests where

f  :: String -> String
f x = show x
  where y :: String
        y = "foo"

PS D:\dev\ws\haskell\cabal-test> hlint .\src\HlintTests.hs --refactor
module HlintTests where

f  :: String -> String
f = show
  where
y = "foo" y :: String

PS D:\dev\ws\haskell\cabal-test> hlint --version
HLint v3.2.1, (C) Neil Mitchell 2006-2020

PS D:\dev\ws\haskell\cabal-test> refactor --version
v0.8.2.1
```

Thanks in advance.

**Comment by zliu41** [comment]
Thanks for reporting the bug! It's fixed in #96. If you'd like a release now please let me know; otherwise I'll make a release after closing #91, #93 and #95, hopefully in the new few days.

**Comment by jneira** [comment]
@zliu41 that was a really prompt response, many thanks
dont worry, i think we could wait to fix all to release
